source/configuration_manual/authentication/proxies.rst

```diff
@@ -39,6 +39,7 @@ addition of proxy field. The common fields to use for both proxying ways are:
   Useful when dividing users across multiple director clusters.
 
 * ``host=s``: The destination server's IP address. This field is required.
+* ``director_tag=s``: The destination tag name.
 * ``source_ip=s``: The source IP address to use for outgoing connections.
```

Comment:
It doesn't really belong to this page, since it's director-specific while this is about proxies in general. We don't really seem to have a good director configuration manual though. https://doc.dovecot.org/configuration_manual/director_settings/ is the closest we have now.